### ISSUE TITLE
Clarify how Supported Proposals work.

### ISSUE THREAD

**Issue by carlsmith** [WebAssembly/wabt#1513]
In the project README, under *Supported Proposals*, it lists the flags for the various proposals, and says the following:

* Proposal: Name and link to the WebAssembly proposal repo
* flag: Flag to pass to the tool to enable support for the feature
* binary: Whether wabt can read/write the binary format
* text: Whether wabt can read/write the text format
* validate: Whether wabt can validate the syntax
* interpret: Whether wabt can execute these operations in wasm-interp or spectest-interp

This implies that the `multi-value` flag (for example) is required for the tools to parse multi-value WAT syntax or to generate the corresponding binary code. However, I am able to compile and run WAT code that uses the multi-value feature, without passing any flags to the tool (obviously, the feature is enabled in my browser).

In short, the README doesn't explain what the flags do very clearly.

**Comment by binji** [comment]
Good point, now that a few of these proposals are standardized, they have been enabled by default in wabt. The README should be updated to make that clearer.

**Comment by carlsmith** [comment]
Ah, that makes sense. Thanks, @binji.

**Comment by carlsmith** [comment]
It makes much more sense now. Thank you, @binji.
